javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -273,14 +273,8 @@ function findWires() {
  * @return {!remote.DriverService} .
  */
 function createWiresService(binary) {
-    // Firefox's Developer Edition is currently required for Marionette.
-  let exe;
-  if (typeof binary === 'string') {
-    exe = Promise.resolve(binary);
-  } else {
-    binary.useDevEdition();
```

Comment:
This is the only line you should remove (forcing users to dev edition for marionette). Users should still be able to opt-in to the dev edition if they want it.